### PR TITLE
[User] F1 UserTable Story and Tests

### PR DESCRIPTION
**Pull request by cstraw2021**
~~- Instructor column to UserTable~~
~~- Toggle admin and toggle instructor buttons to UserTable (only if showToggleButtons is true)~~
~~- Tests for above~~
Above changes now present in PR [#29](https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-2/pull/29)

Storybook with template without toggle buttons:
image

Storybook with template with toggle buttons:
image

Closes https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-2/issues/5 (and #8 if not yet closed)

deployed at https://po-whuang602.dokku-06.cs.ucsb.edu/

### CODE REVIEW

**Review by pconrad — CHANGES_REQUESTED** [review]
The title of this PR is still misleading: you aren't adding the UserTable; it already existed.

Instead, you appear t be modifying it to add some new functionality  Can you summarize what that change in the title?

**Review by vedantshah99 — APPROVED** [review]
lgtm

**Review by whuang602 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM, though it appears that this is now a much smaller PR after merging one of the others; just a minor tweak to the storybook.  Still helpful though!
